PROTOCOL.md

```diff
@@ -2224,17 +2315,20 @@ content-type: application/x-ndjson; charset=utf-8
 }
 ```
 
-## Table Metadata Format
+### Timestamp Format
+Accepted timestamp format by a delta sharing server: `yyyy-[m]m-[d]d hh:mm:ss[.f...]`. The server will use its local time zone to parse the provided timestamp.   
```

Comment:
We discussed previously that it should probably have the timezone in the timestamp format specification?

Reply:
that's not released: https://github.com/delta-io/delta-sharing/commits/main, that's one of the reason I was not putting it in, to add it once released.